test/onnx/test_verification.py

```diff
@@ -1,13 +1,13 @@
 # Owner(s): ["module: onnx"]
 
-import unittest
-
 import torch
 from torch.onnx import _experimental, verification
+from test_pytorch_common import TestCase
```

Comment:
Thanks for the change! Could you instead import
```python
from torch.testing._internal import common_utils
```

(we would also like to import just modules and not the class in general to keep the namespace clean)

and do 

```
class TestVerification(common_utils.TestCase):
```

instead? TestCase was transitively imported in test_pytorch_common and we want to avoid that.

LMK if you think otherwise